cnf-tests/docgen/cmd/fill.go

```diff
@@ -78,21 +78,50 @@ func fill(xmlFile, descriptionsFile string) {
 
 func fillDescriptions(fileName string, tests TestSuite, currentDescriptions map[string]string) error {
 	missing := false
+	removed := false
+	errMissing := "Found tests with no description"
```

Comment:
you can move these error descriptions down, near where you use them. It will make it easier to find them